pytorch_forecasting/metrics/base_metrics.py

```diff
@@ -822,6 +822,9 @@ def mask_losses(self, losses: torch.Tensor, lengths: torch.Tensor, reduction: st
         """
         if reduction is None:
             reduction = self.reduction
+
+        if isinstance(losses, list): losses = losses[0]
```

Comment:
Should that not be something like
`losses = [self.mask_losses(loss, lengths, reduction=reduction) for loss in losses]`?